// _vuepress@0.14.4@vuepress/lib/app/.temp/routes.js
import Vue from 'vue'

import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-7dcb4eb815cb",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/sunmancang/test/yh.github.io/docs/README.md").then(comp => {
        Vue.component("v-7dcb4eb815cb", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-3fc285ebbfca6",
    path: "/about/GithubPages.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/sunmancang/test/yh.github.io/docs/about/GithubPages.md").then(comp => {
        Vue.component("v-3fc285ebbfca6", comp.default)
        next()
      })
    }
  },
  {
    name: "v-b885b80c0774b",
    path: "/about/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/sunmancang/test/yh.github.io/docs/about/README.md").then(comp => {
        Vue.component("v-b885b80c0774b", comp.default)
        next()
      })
    }
  },
  {
    path: "/about/index.html",
    redirect: "/about/"
  },
  {
    name: "v-2dbb01abd72e9",
    path: "/about/VuePress.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/sunmancang/test/yh.github.io/docs/about/VuePress.md").then(comp => {
        Vue.component("v-2dbb01abd72e9", comp.default)
        next()
      })
    }
  },
  {
    name: "v-2560015c1af52",
    path: "/other/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/sunmancang/test/yh.github.io/docs/other/README.md").then(comp => {
        Vue.component("v-2560015c1af52", comp.default)
        next()
      })
    }
  },
  {
    path: "/other/index.html",
    redirect: "/other/"
  },
  {
    name: "v-6cbb38ec2c55d",
    path: "/other/indexTwo.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/sunmancang/test/yh.github.io/docs/other/indexTwo.md").then(comp => {
        Vue.component("v-6cbb38ec2c55d", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
